husky_gazebo/controller/arm_controller_ur5.yaml

```diff
@@ -0,0 +1,29 @@
+arm_controller:
```

Comment:
husky_control is where all robot control configuration and bringup lives, at least anything that is sim and live portable.